vnext/Shared/Networking/WinRTHttpResource.cpp

```diff
@@ -51,6 +51,21 @@ using winrt::Windows::Web::Http::IHttpClient;
 using winrt::Windows::Web::Http::IHttpContent;
 using winrt::Windows::Web::Http::Headers::HttpMediaTypeHeaderValue;
 
+namespace {
+
+constexpr uint32_t operator""_KiB(unsigned long long int x) {
+  return static_cast<uint32_t>(1024 * x);
+}
+
+constexpr uint32_t operator""_MiB(unsigned long long int x) {
```

Comment:
>_MiB [](http://example.com/codeflow?start=29&length=4)

(nit) out of curiosity, why not just _MB?

Reply:
As @aeulitz pointed out in the `main`-facing PR, while conventionally both can be used, `MiB` is slightly more accurate.
`KB` and `MB` **could** refer to `1000` and `1000 * 1000` units.
`KiB`, `MiB`, etc, refer unequivocally to powers of `1024`.

See https://www.majordifferences.com/2018/03/differences-between-megabyte-and.html